### ISSUE TITLE
Button to hide interactive sections of the UI when a demo is currently loaded

### ISSUE THREAD

**Issue by senakafdo** [dsi-icl/gdo-loader#17]
When a demo is loaded, we would only want the control UI relevant to the demo and other parts of the UI can be hidden to avoid accidental interaction with the touchscreen, which will change screens or load new demos with no confirmation.

**Comment by fguitton** [comment]
I fear yet another monkey patching solution for a problem that is inherent to the hardware and that users will probably not end up using anyway because you'd have to be "in-the-know" and it just artificially breaks a user workflow. I perfectly understand there is an issue with the screen, but I'm trying to be realistic as to the effective impact of such a change. People would have to explicitly click to hide, then click to show, then click to hide, it's just not a good thing. You'd make it such a drag to interact with it that no one would actually use it anyway. You're better off asking people not to get too close to the screen the way it's already done. Or alternatively look for a capacitive screen overlay to replace the IR one.

**Comment by senakafdo** [comment]
@fguitton, I fully agree that it is an issue of the screen overlay rather than the software, but we have tried to ask people not to get too closer to the screen and failed on several occasions. Therefore, we need some sort of solution. Users who do not know the existence of such a button will never use it and they would still have the older UX. Users who know that such a button exists would use it prevent any accidental selection of another demo. As with the other buttons, a majority of the frequent users would eventually learn what this does. If we can find a (better) hardware solution, then I am all up for it.
